pkg/ddc/alluxio/cache.go

```diff
@@ -17,94 +17,74 @@ package alluxio
 
 import (
 	"fmt"
-	"regexp"
-	"strings"
 	"time"
 
+	"github.com/fluid-cloudnative/fluid/api/v1alpha1"
 	"github.com/fluid-cloudnative/fluid/pkg/ddc/alluxio/operations"
 	"github.com/fluid-cloudnative/fluid/pkg/utils"
+	"github.com/pkg/errors"
+)
+
+var (
+	cachedPercentageFormat = "%.1f%%"
 )
 
 // queryCacheStatus checks the cache status
 func (e *AlluxioEngine) queryCacheStatus() (states cacheStates, err error) {
-	summary, err := e.reportSummary()
+	// get alluxio fsadmin report summary
+	summary, err := e.GetReportSummary()
```

Comment:
May I know why do you change `reportSummary`  from private to public?

Reply:
because `Convey ` can not mock private function

Reply:
I see. Thanks for explanation.